core/src/main/java/com/microsoft/applicationinsights/telemetry/MetricTelemetry.java

```diff
@@ -28,6 +28,7 @@
 
 import com.google.common.base.Strings;
 import com.microsoft.applicationinsights.internal.util.Sanitizer;
+import org.apache.http.annotation.Obsolete;
```

Comment:
We do not need those imports anymore across all files - you switched everything to `@Deprecated`.

Reply:
@Dmitry-Matveev by all the files do you mean across whole SDK we should remove @Obsoelete? Because there are many files that I haven't touched at all and they contain @Obsolete annotations. I will optimize all imports for the file  I touched so that there are no unused import.